lib/nerves_ssh.ex

```diff
@@ -50,16 +50,34 @@ defmodule NervesSSH do
     GenServer.call(__MODULE__, :info)
   end
 
+  @doc """
+  Add an SSH public key to the authorized keys
+
+  This will also attempt to save the key in `{USER_DIR}/authorized_keys`
```

Comment:
The references to `USER_DIR` feel weird, since I'd expect the user directory for root to be `/root`, the user directory for `jon` to be `/home/jon`, etc. Since the `user` is discarded, maybe the user directory should be `/data/nerves_ssh/default_user`? What do you think?

Reply:
Ya it is odd. Especially since this is acting as an SSH daemon/server. 

I actually think just using `system_dir` here might make more sense. I'm not opposed to `/data/nerves_ssh/default_User` but seems like that might just be overkill

Reply:
I forgot where we ended on this. I know that `~/.ssh` was making testing slightly stressful since `$HOME` was amazingly tricky to override without causing other issues.

I think that I'm at `/data/nerves_ssh/default_user`. Hope that's ok with you.

Reply:
Ya, testing was getting scary because accidentally overwriting `~/.ssh/authorized_keys` would be really bad on a host so I tabled it until I could give more thought.

I'm okay with doing `/data/nerves_ssh/default_user` dir to be on the safe side for implementation and testing